### DISCUSSION TITLE
something about this

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#364] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
this got patch 3 moths ago

**Comment by Meeep1** [comment, marked as answer]
no

**Reply by Aka-but-nice** [reply]
#348 #347

**Reply by ghost** [reply]
Prove it

**Reply by Aka-but-nice** [reply]
Do yaw want documentation

**Reply by Aka-but-nice** [reply]
Sure

**Reply by Aka-but-nice** [reply]
Here watch carefully go to these links tinyurl.com/goofcurly if you have securly, tinyurl.com/goofguardian (broken if you have a updated goguardian)  tinyurl.com/goofclassroom, if you have securly classroom, tinyurl.com/goofsi if you have blocksi and tinyurl.com/goofumbrella if you have cisco umbrella then follow instructions 
[Screen recording 2023-01-08 4.03.15 PM.webm](https://user-images.githubusercontent.com/119691881/211219104-8518c26b-4890-4e8d-a552-6559049c9e17.webm)

**Comment by 3kh0** [comment]
Try it again you bum, you are doing something wrong

**Reply by ghost** [reply]
K owner what to call on discord

**Reply by Aka-but-nice** [reply]
Wut

**Reply by Aka-but-nice** [reply]
try what?

**Reply by Aka-but-nice** [reply]
> K owner what to call on discord

@heys1234 what?

**Comment by obkduishinijf** [comment]
what
